admin/tool/policy/classes/api.php

```diff
@@ -396,4 +402,118 @@ public static function move_up($policyid) {
     public static function move_down($policyid) {
         static::move_policy_document($policyid, 3);
     }
+
+    /**
+     * Returns list of acceptances
+     *
+     * @param int|array $users id of a user or a list of ids
+     * @param int|array $versions list of policy versions
+     * @return array list of acceptances indexed by userid and versionid. Each acceptance has additional field 'policyid'
+     */
+    public static function get_acceptances($versions, $select = '', $params = [], $orderby = 'u.lastname, u.firstname', $limitfrom = 0, $limitto = 0) {
```

Comment:
The signature does not match the docblock. Also, I am not generally a fan of having API designed in a way that it exposes low level details such as SQL statements ...

Reply:
this function will be changed significantly when we add filters for the user list. I don't want to add unittests for it until then

Reply:
Ok